src/games/cclcc/mapsystem.cpp

```diff
@@ -208,7 +208,8 @@ int MapSystemCCLCC::MapLoad(uint8_t* data) {
         : (MapPartsDisp[i].state == Showing || MapPartsDisp[i].state == Shown)
             ? 1 - progress / conversionFactor
             : 0;
-    MapPartsDisp[i].fadeAnim.Direction = inOrOut;
+    MapPartsDisp[i].fadeAnim.Direction =
+        inOrOut ? AnimationDirection::In : AnimationDirection::Out;
```

Comment:
This was previously setting direction to 1 or 0, I'm assuming this was meant to be 1 or -1